Src/HALAL/Models/Packets/SPIPacket.cpp

```diff
@@ -0,0 +1,10 @@
+/*
+ * SpiPacket.hpp
+ *
+ *  Created on: 15 nov. 2023
+ *      Author: ricardo
+ */
+
+#include "HALAL/Models/Packets/SPIPacket.hpp"
+
+map<uint16_t, SPIPacket*> SPIPacket::SPIPacketsByID = {};
```

Comment:
Pro tip: `map<uint16_t, SPIPacket*> SPIPacket::SPIPacketsByID {};`

should work the same